server/src/main/java/org/jboss/as/server/deployment/Phase.java

```diff
@@ -446,6 +446,7 @@ public Phase next() {
     public static final int DEPENDENCIES_MICROPROFILE_METRICS           = 0x1860;
     public static final int DEPENDENCIES_MICROPROFILE_HEALTH            = 0x1870;
     public static final int DEPENDENCIES_MICROPROFILE_OPENTRACING       = 0x1880;
+    public static final int DEPENDENCIES_MICROPROFILE_FAULT_TOLERANCE   = 0x1890;
```

Comment:
@rhusar could you bump the constants please? I'm going to merge #3963 that takes these values.

Reply:
Done; would have needed a rebase anyway.